### PR TITLE
chore: database tab bar ui improvements

### PR DESCRIPTION
**Pull request by richardshiue**
| Before | After |
| :--: | :--: |
| image | image |

### Feature Preview

---

#### PR Checklist

- [x] My code adheres to the [AppFlowy Style Guide](https://appflowy.gitbook.io/docs/essential-documentation/contribute-to-appflowy/software-contributions/submitting-code/style-guides)
- [ ] I've listed at least one issue that this PR fixes in the description above.
- [x] I've added a test(s) to validate changes in this PR, or this PR only contains semantic changes.
- [ ] All existing tests are passing.

### CODE REVIEW

**Comment by annieappflowy** [comment]
Better to show before and after side by side

File: frontend/rust-lib/dart-ffi/Cargo.toml [hunk 1]
```diff
@@ -7,8 +7,8 @@ edition = "2018"
 [lib]
 name = "dart_ffi"
 # this value will change depending on the target os
-# default staticlib
-crate-type = ["staticlib"]
+# default cdylib
+crate-type = ["cdylib"]
```
Comment (LucasXu0) [inline frontend/rust-lib/dart-ffi/Cargo.toml:11]: Don't modify these two lines.

**Comment by richardshiue** [comment]
Is it just me or the theme mode doesn't change 100% of the time?

**Comment by LucasXu0** [comment]
> Is it just me or the theme mode doesn't change 100% of the time?

Do you mean that the theme mode won't change successfully at times?

**Comment by richardshiue** [comment]
> Do you mean that the theme mode won't change successfully at times?

I apologize, it seems like rebasing on main fixed it.

**Pull request merged**